operator/controllers/seldondeployment_controller.go

```diff
@@ -227,6 +227,37 @@ func createIstioResources(mlDep *machinelearningv1.SeldonDeployment,
 			return nil, nil, err
 		}
 	}
+	// Add HTTP routes, if no custom url rewrites get annotation prefix and rewrite or use defaults
+	var routes []*istio_networking.HTTPRoute
+	if mlDep.Spec.UrlRewrites == nil {
+		prefix := getAnnotation(mlDep, ANNOTATION_ISTIO_PREFIX, "/seldon/"+namespace+"/"+mlDep.Name+"/")
+		rewrite := getAnnotation(mlDep, ANNOTATION_ISTIO_REWRITE, "/")
+		mlDep.Spec.UrlRewrites = map[string]string{
+			prefix: rewrite,
+		}
+	}
```

Comment:
Since the default route also gets added through the `UrlRewrites` field - which is part of the editable spec - wouldn't this allow a user to completely remove the default route for Seldon Core? 

These routes are considered part of the standard set of protocols supported by SC, therefore I think they should always be present (even if just on top of custom rewrites).